crates/ty_python_semantic/src/types/diagnostic.rs

```diff
@@ -9,6 +9,7 @@ use crate::types::string_annotation::{
     IMPLICIT_CONCATENATED_STRING_TYPE_ANNOTATION, INVALID_SYNTAX_IN_FORWARD_ANNOTATION,
     RAW_STRING_TYPE_ANNOTATION,
 };
+use crate::types::LintDiagnosticGuard;
 use crate::types::{protocol_class::ProtocolClassLiteral, KnownFunction, KnownInstanceType, Type};
```

Comment:
if you do this then you don't get any line numbers changing in the generated docs :P

```suggestion
use crate::types::{protocol_class::ProtocolClassLiteral, KnownFunction, KnownInstanceType, Type, LintDiagnosticGuard};
```